dtm0/it/all2all/README.md

```diff
@@ -43,3 +43,41 @@ it needs to be patched by the hare.patch in current directory
 to make it able to generate clusted configuration that includes
 DTM0 service. After that it should be built and installed into
 the system (see [README.md](https://github.com/Seagate/cortx-hare/blob/main/README.md) in the root of Hare repo).
+
+## Troubleshooting
+### 1. Test fails for starting motr processes
+If test fails at the stage where it start motr processes, please check if /etc/modprobe.d/lnet.conf has same interface as 
+the lnet service is running on. If interfaces does not match then refer [this](https://github.com/Seagate/cortx-hare/blob/main/README.md#data_iface)
+
+### 2. Test is stuck at starting motr process for first phase
+If the test is stuck at "Starting Motr (phase1, mkfs)... " like shown below, then there is possbility that 
+you have pulled latest code for cortx-motr/dtm0-main branch and rebuilt it, but not rebuilt the hare code again.
+
+```sh
+[root@ssc-vm-c-1966 all2all]# sudo ./all2all
+INFO: Bootstrapping the cluster using Hare...
+2021-04-22 00:12:06: Generating cluster configuration... OK
+2021-04-22 00:12:07: Starting Consul server on this node......... OK
+2021-04-22 00:12:14: Importing configuration into the KV store... OK
+2021-04-22 00:12:14: Starting Consul on other nodes... OK
+2021-04-22 00:12:14: Updating Consul configuraton from the KV store... OK
+2021-04-22 00:12:15: Waiting for the RC Leader to get elected.... OK
+2021-04-22 00:12:17: Starting Motr (phase1, mkfs)...
+
+```
+Solution:
+In case you have pulled latest code, please rebuild motr and hare both. 
+For building motr code again use following command
+```sh
+[root@ssc-vm-c-1966 cortx-motr]# time { MAKE_OPTS=-j32 CONFIGURE_OPTS=--disable-altogether-mode\ --enable-debug\ --enable-dtm0\ --with-trace-ubuf-size=32  ./scripts/m0 rebuild || echo FAIL; }
```

Comment:
No need to use the "time" command.

If it is used, then you need to explain what is the expected result from that command (e.g. how long is a the reasonable amount of time to run such command)

Reply:
ok..